coffeecard/CoffeeCard.Models/DataTransferObjects/v2/Product/AddProductRequest.cs

```diff
@@ -0,0 +1,26 @@
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace CoffeeCard.Models.DataTransferObjects.v2.Product
+{
+    /// <summary>
+    /// Initiate a new product add request
+    /// </summary>
+    public class AddProductRequest
+    {
+        [Required]
+        public int Price { get; set; }
```

Comment:
We should consider a negative integer as an invalid value. Please add a Minumum Data Annotation to check for this. Consider similar checks on NumberofTickets